clients/src/main/java/org/apache/kafka/clients/consumer/NoOffsetForPartitionException.java

```diff
@@ -26,19 +26,20 @@
 
     private static final long serialVersionUID = 1L;
 
-    private final TopicPartition partition;
+    private final Set<TopicPartition> partitionSet;
 
-    public NoOffsetForPartitionException(TopicPartition partition) {
-        super("Undefined offset with no reset policy for partition: " + partition);
-        this.partition = partition;
+    public NoOffsetForPartitionException(String message) {
```

Comment:
I'm a bit confused by this discussion since it doesn't look like this is actually invoked anywhere? Are we missing this in Errors.java or did these comments just grow out of data with updates to the patch? If the latter, shall we remove this constructor?